opus/application/apps/tools/file_utils.py

```diff
@@ -85,10 +85,40 @@ def get_pds_products(opus_id_list,
         sql += ', '+q('obs_files')+'.'+q('checksum')
 
     sql += ' FROM '+q('obs_files')
-    sql += ' WHERE '
+    sql += ' WHERE ('
     if product_types != ['all']:
-        sql += q('obs_files')+'.'+q('short_name')+' IN %s AND '
-        values.append(product_types)
+        # Because we didn't store @version to the database, when multiple versions (or
```

Comment:
in the database
product types
are passed in
query the database
in the product_types list
. The query condition will be